worldguard-bukkit/src/main/resources/plugin.yml

```diff
@@ -3,4 +3,4 @@ main: com.sk89q.worldguard.bukkit.WorldGuardPlugin
 version: "${internalVersion}"
 depend: [WorldEdit]
 softdepend: [CommandBook]
-api-version: 1.13
+api-version: 1.16
```

Comment:
Hmm, never seen people up this beyond 1.13, does this actually do anything or is it just convention?

Reply:
that api-version 1.16 does make the plugin incompatible to 1.15 servers.

e.g. a 1.15 spigot/paper server doesn't load it
> [07:21:05 ERROR]: Could not load 'plugins\worldguard-bukkit-7.0.3-SNAPSHOT-dist.jar' in folder 'plugins'
> org.bukkit.plugin.InvalidPluginException: Unsupported API version 1.16

could be useful for users who misses the labels for the versions. With the Materials changes that plugin isn't compatible with 1.15 anymore.